scep/scep.go

```diff
@@ -382,27 +382,27 @@ func (msg *PKIMessage) DecryptPKIEnvelope(cert *x509.Certificate, key *rsa.Priva
 func (msg *PKIMessage) Fail(crtAuth *x509.Certificate, keyAuth *rsa.PrivateKey, info FailInfo) (*PKIMessage, error) {
 	config := pkcs7.SignerInfoConfig{
 		ExtraSignedAttributes: []pkcs7.Attribute{
-			pkcs7.Attribute{
+			{
```

Comment:
@groob these changes occurred as a result of running code cleanup on this file with intellij. Do you want me to revert it for this PRs?

Reply:
run `gofmt` to get the correct formatting.

Reply:
Done in #140, but looks like github is showing diffs based on an outdated HEAD.